testing/py-geoip-api/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-geoip-api
+_pkgname=geoip-api-python
+pkgver=1.3.2
+pkgrel=0
+pkgdesc="MaxMind GeoIP Legacy Database"
+url="https://pypi.python.org/pypi/GeoIP"
+arch="all"
+license="GPL2"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools geoip-dev"
+checkdepends="py-nose"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/maxmind/$_pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/"$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py2() {
+	replaces="$pkgname"
+	_py python2
+}
+
+_py3() {
+	_py python3
+}
+
+_py() {
+	local python="$1"
+	pkgdesc="$pkgdesc (for $python)"
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+check() {
```

Comment:
Check phase is run after _build_ and before _package_, so please put it between these functions to make abuild more understandable.